packages/sui-studio/bin/sui-studio-deploy.js

```diff
@@ -36,9 +48,11 @@ const now = new NowClient(process.env.NOW_TOKEN)
 
 getSpawnPromise('now', ['rm', deployName, '--yes', '-t $NOW_TOKEN'])
   .catch(() => {}) // To bypass now rm error on the first deploy
+  .then(() => writePackageJson(deployName)) // Add package.json for SPA server
   .then(() => getSpawnPromise(
-    'now', ['./public', '--name=' + deployName, '--static', '-t $NOW_TOKEN']
+    'now', [BUILD_FOLDER, '--name=' + deployName, '--npm', '-t $NOW_TOKEN']
```

Comment:
That work with private repos ?!
I think that is better deploy and static not relative `npm` pages

Reply:
It has to be for SPA as mentionnned by https://zeit.co/blog/now-static
Private package will be implemented later.